languagetool-language-modules/sr/src/main/java/org/languagetool/language/JekavianSerbian.java

```diff
@@ -0,0 +1,72 @@
+/* LanguageTool, a natural language style checker
+ * Copyright (C) 2007 Daniel Naber (http://www.danielnaber.de)
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301
+ * USA
+ */
+package org.languagetool.language;
+
+
+import org.languagetool.rules.Rule;
+import org.languagetool.rules.sr.jekavian.MorfologikJekavianSpellerRule;
+import org.languagetool.rules.sr.jekavian.SimpleGrammarJekavianReplaceRule;
+import org.languagetool.rules.sr.jekavian.SimpleStyleJekavianReplaceRule;
+import org.languagetool.synthesis.Synthesizer;
+import org.languagetool.synthesis.sr.JekavianSynthesizer;
+import org.languagetool.tagging.Tagger;
+import org.languagetool.tagging.sr.JekavianTagger;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ResourceBundle;
+
+
+/**
+ * Class modelling Serbian Jekavian dialect
+ *
+ * @since 4.0
+ */
+public class JekavianSerbian extends Serbian {
```

Comment:
I only noticed this now, but this class has no `getName()` method, is that on purpose?

Reply:
Yes, it is on purpose - no one should use "JekavianSerbian" class directly. This class models Jekavian dialect, that is basic for Serbian language spoken in Montenegro, Bosnia and Herzegovina and Croatia. Dialects are mostly the same, but there are differences between them.

Reply:
Can it be made abstract then maybe, to prevent users from using it directly?

Reply:
I made the class abstract; it will be visible in next pull request.